### ISSUE TITLE
Invalid face reference: python-cell [148 times]

### ISSUE THREAD

**Issue by bamanzi** [thisch/python-cell.el#4]
I got a lot of error messages complaining 'invalid face reference: python-cell'

```
Invalid face reference: python-cell [339 times]
Invalid face reference: python-cell [108 times]
python-shell-send-cellInvalid face reference: python-cell
Invalid face reference: python-cell [62 times]
```

environment: emacs-24.3

It seems that the default value of variable `python-cell-highlight-face` should be `python-cell-highlight-face` rather than `python-cell`

``` diff
--- e:/projects/dotemacs-py/elisp/python-cell.el
+++ #<buffer python-cell.el>
@@ -63,7 +63,7 @@
   "Overlay used by Python-Cell mode to highlight the current cell.")
 (make-variable-buffer-local 'python-cell-overlay)

-(defcustom python-cell-highlight-face 'python-cell
+(defcustom python-cell-highlight-face 'python-cell-highlight-face
   "Face with which to highlight the current cell in Python-Cell mode."
   :type 'face
   :group 'python-cell

```

**Comment by ivanovmg** [comment]
Thank you very much for posting the solution! It works like a charm!

**Comment by thisch** [comment]
Can you come up with a PR. Please

**Comment by thisch** [comment]
Fixed in #7. Closing
